release-notes/6.0/preview/6.0.0-preview.5.md

```diff
@@ -0,0 +1,169 @@
+|  | [Checksums][checksums-sdk]                             | [Checksums][checksums-sdk]                                      | [Checksums][checksums-runtime]                             | [Checksums][checksums-runtime]  | [Checksums][checksums-runtime]  | [Checksums][checksums-runtime]
+
+</br>
+
+1. Includes the .NET Runtime and ASP.NET Core Runtime
+2. For hosting stand-alone apps on Windows Servers. Includes the ASP.NET Core Module for IIS and can be installed separately on servers without installing .NET Runtime.
+
+</br>
+
+The .NET SDK includes a matching updated .NET Runtime. Downloading the Runtime or ASP.NET Core packages is not needed when installing the SDK.
+
+You can check your .NET SDK version by running the following command. The example version shown is for this release.
+
+```console
+$ dotnet --version
+6.0.100-preview.5.21302.13
+```
+
+### .NET Multi-Platform App UI Workload Downloads
+
+.NET 6 introduces Android, iOS, and macOS SDKs for developing native applications. These provide the foundational mobile and desktop pieces for the new [.NET Multi-platform App UI](https://github.com/dotnet/maui). See [dotnet/maui-samples](https://github.com/dotnet/maui-samples) for additional setup instructions and sample projects you can run today.
+
+After installing the .NET SDK, you can install these platform SDKs using the `dotnet workload install` command:
+
+```console
+$ dotnet workload install microsoft-android-sdk-full
+$ dotnet workload install microsoft-ios-sdk-full
+$ dotnet workload install microsoft-maccatalyst-sdk-full
+$ dotnet workload install microsoft-macos-sdk-full
+$ dotnet workload install microsoft-tvos-sdk-full
+```
+
+</br>
+
+Visit [.NET Documentation](https://docs.microsoft.com/dotnet/core/) to learn about .NET, for building many different types of applications.
+
+
+## Docker Images
+
+The [.NET Docker images](https://hub.docker.com/_/microsoft-dotnet) have been updated for this release. The [.NET Docker samples](https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md) show various ways to use .NET and Docker together. You can use the following command to try running the latest .NET 6.0 release in containers:
+
+```console
+docker run --rm mcr.microsoft.com/dotnet/samples
+```
+
+The following repos have been updated.
+
+* [dotnet/sdk](https://hub.docker.com/_/microsoft-dotnet-sdk/): .NET SDK
+* [dotnet/aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet/): ASP.NET Core Runtime
+* [dotnet/runtime](https://hub.docker.com/_/microsoft-dotnet-runtime/): .NET Runtime
+* [dotnet/runtime-deps](https://hub.docker.com/_/microsoft-dotnet-runtime-deps/): .NET Runtime Dependencies
+* [dotnet/samples](https://hub.docker.com/_/microsoft-dotnet-samples/): .NET Samples
+
+
+## Visual Studio Compatibility
+
+You need [Visual Studio 17.0](https://visualstudio.microsoft.com) or later to use .NET 6.0 on Windows. 
```

Comment:
should we spell it out like in the release notes?
https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes

```suggestion
You need [Visual Studio 2019 version 17.0](https://visualstudio.microsoft.com) or later to use .NET 6.0 on Windows. 
```